prebuilt-ui/basic-embed/pages/index.js

```diff
@@ -0,0 +1,24 @@
+import PrebuiltCall from '../components/PrebuiltCall';
+import getDemoProps from '@dailyjs/shared/lib/demoProps';
+
+export default function Index({ isConfigured = false }) {
+  return (
+    <>
+      <PrebuiltCall configured={isConfigured} />
+      <style jsx>{`
+        display: flex;
```

Comment:
not sure how this styling works without a selector... maybe not needed?

Reply:
Huh, I thought I needed it, but maybe not. I borrowed from what's happening [in the basic-call demo](https://github.com/daily-demos/examples/blob/0829203d78a8d0bafb34bfa8cff0940c8bdedf79/dailyjs/basic-call/pages/index.js#L98). 

From playing with this and adding the `inCall` wrapper, I can't pinpoint why but my mobile styles aren't working any more 🙃 . I'll have to play with the CSS on this a bit more and look at our other demos again.

Reply:
I think you'd still need to add a selector though.. I'm not sure what these styles are getting applied to

Reply:
I also was not positive, so now I've got a selector in there! Thanks again.